src/frontend/src/sass/_table.scss

```diff
@@ -41,4 +41,8 @@
       }
     }
   }
+
+  .link {
+    padding-top: 1.2em;
+  }
```

Comment:
Now that you've removed the button, you can just replace line 38 with `& > .link` and get rid of these lines. And you should be good to go.

Reply:
but the problem is `link` does not follow the BEM naming convention. So, you could instead do
`& > a` and remove `link` class altogether from the <Link> element

Reply:
make the change, please check again

Reply:
I meant just drop in replacement for button to `a`. Let me suggest a commit. Let me know if that makes more sense to you.